ctapipe/tools/tests/test_tools.py

```diff
@@ -11,13 +11,13 @@
 
 def test_muon_reconstruction(tmpdir):
     from ctapipe.tools.muon_reconstruction import MuonDisplayerTool
-    tool = MuonDisplayerTool()
-    tool.run(
+    return_code = MuonDisplayerTool().run(
         argv=shlex.split(
-            f'--events={GAMMA_TEST_LARGE} '
+            f'--input={GAMMA_TEST_LARGE} '
             '--max_events=2 '
         )
     )
+    assert return_code == 0
 
     with pytest.raises(SystemExit):
         tool.run(['--help-all'])
```

Comment:
this seems to be failing (it could be a line that is there due to a bad merge, since `tool` doesn't exist in the new version)